control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -2589,9 +2591,22 @@ components:
           type: object
           additionalProperties:
             type: string
+        affinitykey:
```

Comment:
I wonder if we should split the topology into requested and actual topology.
Then on request for volume creation affinityKey is a vector, but on the actual volume, then it's a dict of key:value
Thoughts?  CC @sinhaashish  @Abhinandan-Purkait